config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```diff
@@ -55,9 +55,9 @@ presubmits:
           - ./scripts/ci-e2e.sh
         env:
           - name: GINKGO_FOCUS
-            value: "Workload cluster creation"
+            value: \[REQUIRED\]
```

Comment:
for this to work we need to either 1) backport the capz PR to release-1.1, or 2) wait until we release 1.2 in a couple of days to merge this

Reply:
I vote for a cherry-pick, just so I don't forget to do this. Created:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2119